packages/connect/src/api/common/paramsValidator.ts

```diff
@@ -147,6 +147,7 @@ export const getFirmwareRange = (
             }
             if (c.coin) {
                 // rule for coin shortcut
+                // @ts-expect-error
                 return (typeof c.coin === 'string' ? [c.coin] : c.coin).includes(shortcut!);
```

Comment:
Can you verify @mroz22 got this error:

```
Property 'coinType' does not exist on type '{ coin: string[]; methods: string[]; min: string[]; max: undefined; comment: string[]; capabilities?: undefined; } | { coin: string[]; min: string[]; comment: string[]; methods?: undefined; max?: undefined; capabilities?: undefined; } | { methods: string[]; min: string[]; comment: string[]; coin?: undefined; max?: undefined; capabilities?: undefined; } | { capabilities: string[]; min: string[]; comment: string[]; coin?: undefined; methods?: undefined; max?: undefined; } | { coin: string[]; methods: string[]; min: string[]; comment: string[]; max?: undefined; capabilities?: undefined; } | { capabilities: string[]; methods: string[]; min: string[]; coin?: undefined; max?: undefined; comment?: undefined; }'.
  Property 'coinType' does not exist on type '{ coin: string[]; methods: string[]; min: string[]; max: undefined; comment: string[]; capabilities?: undefined; }'
```

It seems it could be related to some missing protobuf definition as mentioned in comments above

Reply:
I see `Unused '@ts-expect-error' directive.` and type-check pass fine for me locally without the ts-expect-error 🤷 
![image](https://user-images.githubusercontent.com/3729633/202475323-dc05a7cb-3dbd-4bfa-812f-e33f234f7957.png)

Reply:
@matejkriz Please try `yarn && yarn tsc --version` that you are really on version 4.9 - because if it really was unused it should also throw on CI which is passing right now

Reply:
Now I see that it doesn't throw on `yarn run type-check` where tsconfig.json is used, but it does throw on build:lib where tsconfig.lib.json is used

Reply:
I had old TS version in VS code selected, so VS code warning is gone, but still it doesn't throw for me on yarn type-check even without @ts-expect-error 🤷  let's call it my local issue, feel free to merge!